### ISSUE TITLE
PhpStorm 2016.3.2 fails to start: NoSuchMethodError: getOverrideLafFonts

### ISSUE THREAD

**Issue by DuckThom** [ChrisRM/material-theme-jetbrains#312]
I'm using @mallowigi's fork, version 0.3.1 but since I cannot make issues there, I'll make one here.
After enabling the plugin on PhpStorm 2016.3.2 I get the following message when starting PS.

```
Plugin 'com.chrisrm.idea.MaterialThemeUIFork' failed to initialize and will be disabled.  Please restart PhpStorm.

java.lang.NoSuchMethodError: com.intellij.ide.ui.UISettings.getOverrideLafFonts()Z
    at com.chrisrm.idea.MTTheme.activate(MTTheme.java:92)
    at com.chrisrm.idea.MTColorScheme.initComponent(MTColorScheme.java:10)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:501)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:125)
    at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:444)
    at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
    at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:451)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:109)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:403)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:389)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.intellij.idea.MainImpl$1$1.a(MainImpl.java:69)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

**Comment by Alex-Mann** [comment]
Also getting this error in Webstorm 2016.3.3

**Comment by paulpooch** [comment]
@Alex-Mann 
Yep - same in:

WebStorm 2016.3.3
Build #WS-163.12024.17, built on January 31, 2017

**Comment by ColinetCyril** [comment]
@paulpooch Same issue on all my jetbrains suit (webstorm, intellij IDEA, CLion, RubyMine and AndroidStudio)  :/
(Surely due to the recent update of Material UI Fork...)

```IDE: PhpStorm (build #PS-163.10504.2, 20 Dec 2016 15:21) 
OS: Windows 10 (10.0, amd64) 
JRE: 1.8.0_91-b14 (Oracle Corporation) 
JVM: 25.91-b14 (Java HotSpot(TM) 64-Bit Server VM)```

**Comment by mallowigi** [comment]
Guys, I've fixed the issue in my latest release (v0.3.1.3 for EAP users, 0.3.1.2 for the others). This also allows the tab border to be on the right side according to the tabs placement.

Please update.
